dffml/secret/file.py

```diff
@@ -0,0 +1,46 @@
+import os
+import hashlib
+import asyncio
+import pathlib
+from typing import Union,Any
+
+from ..base import config
+from ..util.entrypoint import entrypoint
+from .base import BaseSecretContext, BaseSecret
+
+from dffml.high_level import Record
+from ..source.memory import MemorySourceContext
+from ..source.ini import INISourceConfig,INISource
+
+
+
+class FileSecretContext(BaseSecretContext, MemorySourceContext):
+    def __init__(self,*args,**kwargs):
+        super().__init__(*args,**kwargs)
+
+
+    async def get(self, name: str) -> Union[None, str]:
+        with open(self.parent.config.filename,'r+') as fd:
+            await self.parent.load_fd(fd)
+        record = self.parent.mem.get(name,None)
```

Comment:
I'm also realizing that this will end up with `.ini` files that look like:

```ini
[thing0]
data = data0

[thing1]
data = data1

[thing2]
data = data2
```

Whereas we would probably be better off doing:

```ini
[secrets]
thing0 = data0
thing1 = data1
thing2 = data2
```